lib/index.js

```diff
@@ -76,6 +68,9 @@ export function activate() {
   subscriptions.add(atom.config.observe('linter-stylelint.showIgnored', (value) => {
     showIgnored = value;
   }));
+  subscriptions.add(atom.config.observe('linter-stylelint.grammarScopes', (value) => {
+    grammarScopes = value;
```

Comment:
This setting needs to be removed, but just for your reference the implementation here wouldn't have worked anyway.

A reference to the _initial_ `grammarScopes` down in the return from `provideLinter` is the only one that is ever used, so you would have needed to update the current Array not replace it as you are doing here.